.github/workflows/test.yaml

```diff
@@ -566,13 +566,14 @@ jobs:
       fail-fast: false
       matrix:
         node:
-          - 14 # latest minor+patch oh GH Actions
-          - 15 # latest minor+patch oh GH Actions
-          - 16 # latest minor+patch oh GH Actions
+          - 14 # latest minor+patch on GH Actions
           - 14.17.0 # minimal node version via https://github.com/prisma/prisma/blob/main/packages/client/package.json and minimal minor+patch version of node 14 via https://www.prisma.io/docs/reference/system-requirements
+          - 15 # latest minor+patch on GH Actions
+          - 16 # latest minor+patch on GH Actions
           - 17
```

Comment:
Node.js 17 is neither an LTS nor a maintenance version of Node.js.
See https://github.com/nodejs/release#release-schedule.

```suggestion
```